test/jdk/java/security/spec/RC2ParameterSpec/InvalidArrayIndex.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
```

Comment:
Copyright is not correct, see other open tests for copyright to copy.

Reply:
Fixed. How did that happen.